ui/ui-community/tsconfig.json

```diff
@@ -14,7 +14,8 @@
     "isolatedModules": true,
     "jsx": "preserve",
     "experimentalDecorators": true,
-    "emitDecoratorMetadata": true
+    "emitDecoratorMetadata": true,
+    "incremental": true
```

Comment:
I'll need to double check this, but it may break builds across multiple systems. May be fine if it isn't committed.

Reply:
This was turned on by Next automatically (not by me) - they seem pretty sure they want it on 🤷

Reply:
Good enough for me!